### PR TITLE
Add our fellow Americans 😃🗽🇺🇸🏈🍔

### PR DESCRIPTION
**Pull request by dmcrodrigues**

### CODE REVIEW

**Review by viorel15 — APPROVED** [review]

**Review by brocoo — APPROVED** [review]
🗽🇺🇸🏈🍔

**Review by RuiAAPeres — APPROVED** [review]

**Review by diegopetrucci — COMMENTED** [review]
Should we add ios-usa-support as the public facing channel? Or is there a better one?

**Comment by dmcrodrigues** [comment]
That one is quite specific to iOS so not sure if we should include it. Let's check if there's a better one but I'm not aware of one.

File: README.md [hunk 1]
```diff
@@ -51,29 +51,33 @@ At Babylon, we firmly believe that **transparency** is a core value that should
...
+| Prescriptions                 | Adam, Konrad                         |    2/2       | [#prescription-squad](https://babylonhealth.slack.com/archives/C88TCM9JB) |
+| Healthcheck                   | Ben, Catarina, Julien                |    3/3       | [#healthcheck_tribe](https://babylonhealth.slack.com/archives/C7995CX3R) |
+| Native/Core                   | Giorgos, Jason, Martin, Inami, Emese |    5/6       | [#nativeapps_public](https://babylonhealth.slack.com/archives/CE5P8LRNH) |
+| Professional Services         | Chitra                               |    1/3       | [#telus](https://babylonhealth.slack.com/archives/CAJ7YQZ5Z) |
+| Enrolment and Integrity       | Diego                                |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
+| Core Experience               | Sergey, Yuri                         |    2/2       | [#core-experience](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
+| Health Management             | David, João                          |    2/2       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
```
Comment (mluisbrown) [inline README.md]: @dmcrodrigues you are no longer in Health Management, are you?

**Review by mluisbrown — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dmcrodrigues) [reply README.md]: I'm not, I'll open a follow up to change this and include @AnnKatF.

**Comment by dmcrodrigues** [comment]
@diegopetrucci added, thanks!

**Pull request merged**